benchmarks/buiter_et_al_2016_jsg/exp_2_high_resolution.prm

```diff
@@ -0,0 +1,210 @@
+
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function expression = if((x<=0.35 & y<=0.012 & y>0.002) || (x<=0.35 & y>0.022 & y<=0.032), 1,0); \
+                              if((x<=0.35 & y>0.012 & y<=0.022), 1,0); \
+                              if((x>0.35 & x<=0.352 & y>0.002) ||(y<=0.002), 1, 0); \
+                              if(x>0.352 & y>0.002,1,0); \
+                              if(x<=0.35 & y>0.032,1,0); 0
+  end
+end
+
+# Temperature boundary conditions
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+  set List of model names    		      = initial temperature
+end
+
+# Initial temperature field
+subsection Initial temperature model
+  set Model name 			      = function
+  subsection Function
+    set Function expression 		      = 293
+  end
+end
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Zero velocity boundary indicators	      = bottom		# bottom - no slip
+  set Tangential velocity boundary indicators = left		# left - free slip
+
+## right - material inflow with a velocity of 2.5 cm/hour at the height of the pushing block,
+## the velocity is linear decreased to 0 cm/hour at the base of the model.
+  set Prescribed velocity boundary indicators = right:function 
+    subsection Function
+      set Variable names      		      = x,y
+      set Function constants  		      = cm=0.01, h=3600, th=0.002
+      set Function expression 		      = if(y>th, -2.5*cm/h, -(y/th)*2.5*cm/h); 0
+   end
+end
+
+## For top boundary condition, zero traction ("open" zero normal stress) and a sticky air layer 
+## is used to approximate a free surface. Additional testing revealed that using a true free 
+## surface leads to significant mesh distortion and associated numerical instabilities.
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top: zero traction
+end
+
+# Use discontinous composition bound preserving limiter
+subsection Discretization
+  set Use discontinuous composition discretization = true
+  subsection Stabilization parameters
+      set Use limiter for discontinuous composition solution = true
+      set Global composition maximum 	      = 1, 1, 1, 1, 1, 100
+      set Global composition minimum 	      = 0, 0, 0, 0, 0, 0
+  end
+end
+
+# Material properties
+subsection Material model
+
+## Using harmonic material averaging is required to achieve reasonable convergence 
+## behavior, particulally when the viscosity contrast is large.
+  set Material averaging 		      = harmonic average	
+
+  set Model name 			      = visco plastic
+
+  subsection Visco Plastic
+
+    set Reference temperature 		      = 293
+    set Reference viscosity 		      = 1e8
+    set Minimum strain rate 		      = 1e-20
+    set Reference strain rate 		      = 2e-5
+    set Minimum viscosity 		      = 1e5  # The viscosity contrast is 10^7 here.
```

Comment:
```suggestion
    set Minimum viscosity 		      = 1e5 
```

Reply:
Can you put a space between this line and the line above `set Reference strain rate` then add a comment above `set Minimum viscosity` that describes what happens if you change the viscosity contrast between the Minimum and Maximum (e.g., what values work verse don't work?)?